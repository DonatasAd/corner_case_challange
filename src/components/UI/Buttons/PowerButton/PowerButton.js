import React from 'react';
import Power from '../../../../assets/power-button.png';
import styles from './PowerButton.module.css';

export default function PowerButton(props) {
  return (
    <button {...props} className={styles.Button}>
      <img src={Power} className={styles.Logo} alt='' />
    </button>
  );
}

PowerButton.defaultProps = {
  onClick: () => console.log('Power button'),
};