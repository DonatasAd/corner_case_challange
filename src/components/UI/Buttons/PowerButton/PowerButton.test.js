import React from 'react';
import { shallow } from 'enzyme';
import PowerButton from './PowerButton';

describe('PowerButton', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<PowerButton  />);
    expect(component).toMatchSnapshot();
  });
});