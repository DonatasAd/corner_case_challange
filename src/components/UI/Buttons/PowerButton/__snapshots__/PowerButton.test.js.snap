// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PowerButton should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
